app/code/Magento/InventoryGroupedProduct/Plugin/OutOfStockFilter.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
This plugin seems to be pretty much the same as `\Magento\GroupedCatalogInventory\Plugin\OutOfStockFilter` meaning that both of them would be now executed on each operation. This might have a noticeable effect on performance and generally is not the most optimal design.